WORKSPACE

```diff
@@ -36,10 +36,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelrc and .bazelversion files.
 #
-# envoy-wasm commit date: 10/10/2019
-ENVOY_SHA = "10546ebcef86aa8fc25ff08dcd11196988344afc"
+# envoy-wasm commit date: 10/11/2019
+ENVOY_SHA = "c152ba0e19f6536bd604da782994536ad73a8164"
```

Comment:
Sorry, could you update again to pull https://github.com/envoyproxy/envoy-wasm/pull/246 (bugfix)?

Reply:
Hey, this is not the last proxy change.

Reply:
Done

Reply:
I know, but this change would pull the bug that I introduced yesterday, and fixed in the aforementioned PR. Thanks!